python/requirements.txt

```diff
@@ -48,7 +48,7 @@ scikit-image==0.21.0
 prometheus_client>=0.7.1
 pandas
 tensorboardX
-aiohttp>=3.7,!=3.9.0
+aiohttp>=3.7,<3.10.0
```

Comment:
you might still want to keep the `!=3.9.0` at the same time?

Reply:
I am assuming the `!=3.9.0` was there for some reason..

Reply:
you also need to update the `setup.py`

Reply:
it used to be `<3.9.0` and I changed it to `!=3.9.0`; nothing specific about `3.9.0`, `<3.10.0` is properly fine (this file is only used to build the ray docker image)